client/analytics/report/taxes/config.js

```diff
@@ -52,15 +58,29 @@ export const filters = [
 			{ label: __( 'All Taxes', 'wc-admin' ), value: 'all' },
 			{
 				label: __( 'Comparison', 'wc-admin' ),
-				value: 'compare-tax-codes',
+				value: 'compare-taxes',
 				chartMode: 'item-comparison',
 				settings: {
 					type: 'taxes',
 					param: 'taxes',
-					getLabels: getRequestByIdString( NAMESPACE + '/taxes', tax => ( {
-						id: tax.id,
-						label: getTaxCode( tax ),
-					} ) ),
+					getLabels: function( queryString = '' ) {
+						const idList = getIdsFromQuery( queryString );
+						if ( idList.length < 1 ) {
+							return Promise.resolve( [] );
+						}
+						const payload = stringifyQuery( {
+							per_page: -1,
```

Comment:
I don't know if `per_page` works with -1. The API does validation and sets this between 1-100:

https://github.com/woocommerce/woocommerce/blob/d0491072e8a7d0a4535db0ec2bae4a6d43ff64fd/includes/api/v1/class-wc-rest-taxes-controller.php#L665-L673.

Long term, I think we should probably update the API to have an `include` param. That would be consistent with the other filters. That would be a bit more work on top of this PR though. Maybe this is OK for now / initial release of the feature plugin (capped at 100 tax rates via comparison which seams reasonable since you can only chart 5 at once anyway). 

tl;dr I'm OK with maybe just adding a TODO for now about adding an `include` param and the limit here and going with this.

Reply:
@justinshreve I think `apiFetch` handles `-1` in batches so this should be okay.  https://github.com/WordPress/gutenberg/blob/1020543c18a7e303e652ba5b27c4e434642ec72c/packages/api-fetch/src/middlewares/fetch-all-middleware.js#L39

But I do agree that long-term it's a good call to have an `include` param instead of fetching all.  

@psealock I think we can handle that in a separate PR if you'd like and I can reword this issue to address the include param - https://github.com/woocommerce/wc-admin/issues/1640

Reply:
I have an open PR that will add the `include` param for use here - https://github.com/woocommerce/wc-admin/pull/1672